contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -995,6 +1014,12 @@ void Init_cext()
     Trilogy_TimeoutError = rb_const_get(Trilogy, rb_intern("TimeoutError"));
     rb_global_variable(&Trilogy_TimeoutError);
 
+    Trilogy_BaseConnectionError = rb_const_get(Trilogy, rb_intern("BaseConnectionError"));
```

Comment:
I should have given this feedback in the previous PR 😓, but should we rename this `ConnectionError`? I find the "Base" prefix a little out of place as nothing inherits this.